include/seqan3/alignment/pairwise/execution/execution_handler_sequential.hpp

```diff
@@ -59,7 +60,36 @@ struct execution_handler_sequential
         static_assert(std::ranges::view<first_range_type>, "Expected a view!");
         static_assert(std::ranges::view<second_range_type>, "Expected a view!");
 
-        delegate(func(idx, std::move(first_range), std::move(second_range)));
+        delegate(algorithm(idx, std::move(first_range), std::move(second_range)));
+    }
+
+    /*!\brief Takes underlying range of sequence pairs and invokes an alignment on each instance.
+     * \tparam fn_type           A function which performs the alignment.
+     * \tparam job_range_type    The type of underlying sequence pairs; must model std::ranges::view.
+     * \tparam delegate_type     The type of the callable invoked on the std::invoke_result of `fn_type.
+     *
+     * \param[in] algorithm    The callable which will invoked the alignment algorithm.
+     * \param[in] job_range    The range of underlying sequence pairs to be aligned.
+     * \param[in] delegate     The callable which will be invoked on the result of the alignment, writing it to previously (captured) allocated space.
```

Comment:
~The~ A callable which will be invoked on ~the~ each result of the **computed** alignment**s** ~, writing it to previously (captured) allocated space~.